popSimManu.tex

```diff
@@ -514,7 +538,8 @@ \subsection*{Use case: comparing methods of demographic inference}
 For simplicity, we conducted inference in \dadi and \fastsimcoal by fitting an IM model
 with constant population sizes and bi-directional migration \citep{hey2004im}. Our motivation for using
 an IM model was to mimic the approach often used on empirical datasets, where a relatively
-simple model is fit that may not reflect the true underlying demography.
+simple model that may not reflect the true underlying demography is fit.
+% ALG: Sounds better to me like that - but if someone doesn't agree grammatically, put it back.
```

Comment:
i prefer the way it is written currently

Reply:
awkward but ok